mayhem/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM bcapuano/atheris:2.1.1-python3.10
```

Comment:
The single pip install command should be sufficient, could you elaborate?

Reply:
it was left over, ive removed it